### PR TITLE
No job listings content

### PR DESCRIPTION
**Pull request by robbiepaul**
# User need
> As a user
> When I visit the service and there are no job listings
> So that I know to revisit the service in future
> I want to know that this is an expected slow point

# Description
The content is time specific and only applicable while we're in summer holidays. We may want to refactor the code to have a toggle to flip this message off, however I thought I'd raise the PR as this is the minimum that meets the needs on the card.

# Screenshot
image

### CODE REVIEW

**Comment by robbiepaul** [comment]
[No job listings content explains to users that this is expected](https://trello.com/c/HCpLtbx8/291-no-job-listings-content-explains-to-users-that-this-is-expected)

File: app/views/vacancies/index.html.haml [hunk 1]
```diff
@@ -19,8 +19,16 @@
...
           = render partial: 'vacancy', locals: { vacancy: vacancy }
 
-    - else
+    - elsif @vacancies.searched
       %div.empty
         = t('jobs.no_jobs')
+    - else
+      %div.empty
+        %p
+          This is because schools are on summer holidays and this isn't a peak period in the teacher
```
[the remark this answers is not in the document]
Comment (robbiepaul) [reply app/views/vacancies/index.html.haml]: Sure. What content should we display if we turn the ENV variable off? Usually I'd expect to find these details on the card

[the remark this answers is not in the document]
Comment (robbiepaul) [reply app/views/vacancies/index.html.haml]: I'm curious why you think the ENV toggle is necessary? For me it feels the amount of developer effort to add a ENV variable is greater than changing the locale file later

[the remark this answers is not in the document]
Comment (tahb) [reply app/views/vacancies/index.html.haml]: A toggle would be nice to have but it's very important we ship this as soon as we can given the current state of listings.

I'm confident in the team that we will remember to iterate this again in the same way we'd have to remember to toggle it off.

Let's continue.

**Pull request merged**